lib/cli/src/frameworks/react/js/Header.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Header } from './Header';
+
+export default {
+  title: 'Example/Header',
+  component: Header,
+};
```

Comment:
Did we want to add `argTypes` action annotations or a `argTypeRegexp` parameter here?

Reply:
I guess in the template I ended up putting it in `preview.js` after some discussion. Do we want to do that here too? 

@yannbf mentioned that colleagues found it confusing when the docs mention putting stuff in `preview.js` (which you almost inevitably need to do in configuring SB), but the file isn't there to begin with.

OTOH there is something nice about having a minimal set of configuration files in place when you start and adding to them as you get more complex in your usage.

Reply:
https://github.com/chromaui/storybook-template-react/blob/c7eb09bf49da39b5e25226e7f6a69ba3a2b880af/.storybook/preview.js#L3

Reply:
Thanks for the quick review @tmeasday — will update!

Reply:
> Do we want to do that here too?

It's already done when we introduced essentials in the CLI. 
In my opinion, I prefer having it in a single place and never think of it again :)